sdk/keyvault/Azure.Security.KeyVault.Administration/src/BackupRestoreResult.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Security.KeyVault.Administration
+{
+    /// <summary>
+    /// The result of a backup or restore operation.
+    /// </summary>
+    public abstract class BackupRestoreResult
```

Comment:
As mentioned above, I really don't think this is worth it. Autorest-generated code doesn't create abstractions like this (just for discriminators) and the duplication here is trivial.

Reply:
...besides, your summaries below become wrong for `RestoreResult`. The abstraction already creates issues. Fixing that means something like "backup or restore operation", but the it looks weird on each of those distinct classes.

Reply:
The main reason for the duplication is so that I can use it as a base type across the two restore operations, which are functionally identical except for name.